src/components/partials/EventFlyout.vue

```diff
@@ -334,8 +334,9 @@ export default defineComponent({
 
       const isClickOutside = !flyout.contains(e.target);
       const isClickOnEvent = !!e.target.closest('.is-event');
+      const closeOnClickOutside = this.config.eventDialog?.closeOnClickOutside ?? true;
 
-      if (this.isVisible && isClickOutside && !isClickOnEvent) {
+      if (this.isVisible && isClickOutside && !isClickOnEvent && closeOnClickOutside) {
```

Comment:
Test case for EventFlyout.test.ts:

`it("should not close flyout on clicking, when disabled through config", ...)`